### ISSUE TITLE
Needs Review before Launch

### ISSUE THREAD

**Issue by nothingismagick** [tauri-apps/tauri-registry#1]
This is a rescued project, and needs to be reviewed for security and up-to-date-ness.

**Comment by Beanow** [comment]
Are these for mirrors? That sounds interesting.

**Comment by nothingismagick** [comment]
Yes, the idea is that we do all of our "work" here in github in public as usual, but maintain our own mirrors so that our software is not locked up in a garden or unavailable because of national firewalls / general internet outages.

The Verdaccio also brings some other security benefits, but that is a much different issue.

**Comment by Beanow** [comment]
https://docs.gitea.io/en-us/comparison/ may be a better option than gitlab for mirroring. It should be much lighter on server resources. Especially if intended as a read-only one.

**Comment by Beanow** [comment]
1. Some early feedback, versions should be updated.

https://github.com/tauri-apps/tauri-registry/blob/15dde134782431522725544b0ff80f1255393d7c/Vagrantfile#L11

https://github.com/tauri-apps/tauri-registry/blob/15dde134782431522725544b0ff80f1255393d7c/Vagrantfile#L39

2. Using a VM as base, should probably include firewall and fail2ban.

3. Strongly encourage replacing `jwilder/nginx-proxy` with https://traefik.io/ for discovery, routing, load-balancing and let's encrypt / tls termination needs.

**Comment by Beanow** [comment]
More a question, verdaccio uses gitlab for it's auth. Does that make sense when gitlab (or replacement) is a passive mirror?
